Customer Support Procedures/LKE-v4.1-migration.md

```diff
@@ -105,6 +106,23 @@ This change simplify the logs configuration.
 The default log level in the `data/config/logger.json` file is changed to `info`.
 If needed, it can be changed to another value once Linkurious enterprise is updated to v4.1.
 
+# Worker pool
+The worker pool has been introduced to handle heavy computation tasks by offloading it from the main Linkurious enterprise process.
+Currently, the worker pool is used for running alerts, custom queries and visualization layouts.
+
+***Impacted clients:***
+
+- Every customer
+
+***Benefits:***
+
+- Enhanced the responsiveness of the application by offloading heavy computation tasks to separate threads.
+
+***Impacts:***
+
+- The configuration `advanced.layoutWorkers` has been removed from the configuration file as the layout computation is now handled by the worker pool.
+
```

Comment:
The worker pool is disabled by default for existing customers. We may want to point them to the documentation, so that they easily find how to enable it:

```suggestion

See how to enable and configure the worker pool in [the Linkurious Enterprise documentation](https://doc.linkurious.com/admin-manual/latest/advanced-settings/#worker-pool-settings).
```